components/list/list.js

```diff
@@ -0,0 +1,60 @@
+// Imports
+import React, { PropTypes } from 'react';
+import { getClassNamesWithMods } from '../_helpers';
+
+/**
+ * General List component. Use when you need to display array of elements
+ */
+function List({ items, hideBullets, mods = [], align }) {
+  mods.push(`align_${align}`);
```

Comment:
I am happy to quickly discuss it tomorrow.

@AlexDudar: I think you can merge the PR as is. the reference issue can be taken care of later along with all the other components.